### ISSUE TITLE
Need help with something

### ISSUE THREAD

**Issue by splosions** [xyzz/openmw-android#23]
i can't access openmw.cfg and settings.cfg using es file explorer and es file explorer pro, when i try to access it says there is nothing with cfg, i searched all my phone and i found nothing with cfg, except when I hit openmw's own app and saw that there's something like openmw.cfg inside it, but inside there's nothing and a error every time I try to touch it.
I do not know what to do, the game is working but it has a bit of lag, and I can not see the water (except when it goes all white), I wanted to try to download an optimization mod and maybe get it fixed the water
but I just can't do anything, and I don't even know what to try, can someone please help me?

**Comment by licaon-kter** [comment]
Total Commander works fine


/close as invalid

**Comment by splosions** [comment]
total commander is not working either

**Comment by licaon-kter** [comment]
Ok, so you have an issue with your device, what device? What Android version? Did you grant storage permission to the apps? Pics of the errors in Total Commmander?

**Comment by splosions** [comment]
how can i send images?
 i have a screenshot here

**Comment by licaon-kter** [comment]
There's an upload button when you write, you can drag and drop an image, you can host it on imgur.com and link here etc

**Comment by splosions** [comment]
![Screenshot_2019-09-24-17-56-02](https://user-images.githubusercontent.com/51303463/65575278-d32a1a00-df45-11e9-8b5e-1a0efe35168a.png)



translating: the application cannot be installed

**Comment by licaon-kter** [comment]
And? GEt the armv7 version or the universal one...this is not a OpenMW issue...fix whatever Android issue you have in general, then come back

Try with an editor app: https://search.f-droid.org/?q=editor&lang=en

**Comment by splosions** [comment]
Text editor app?

**Comment by licaon-kter** [comment]
@splosions I just linked you to a bunch of editors.

Anyvay, the white water, with Shaders on & Refraction off? 

Pics?

**Comment by splosions** [comment]
without reflexion

![Screenshot_2019-09-26-02-27-36](https://user-images.githubusercontent.com/51303463/65660759-13e96800-e006-11e9-8239-c7a297bb1c74.png)





with reflexion

![Screenshot_2019-09-26-02-28-15](https://user-images.githubusercontent.com/51303463/65660788-2c598280-e006-11e9-8edb-ab5265afb3c9.png)

**Comment by licaon-kter** [comment]
This is build 25?  What device? Android version?

**Comment by splosions** [comment]
Android version 5.1.1,
samsung galaxy j3 (2016) and
I don't know what build :/

**Comment by licaon-kter** [comment]
It's the latest from F-Droid? It's the Nightly?

**Comment by splosions** [comment]
i'm using the nightly and use the glesv2 crash the app

**Comment by licaon-kter** [comment]
So there seems to be an issue with your device and drivers, not something that OpenMW can fix though.

Is 5.1.1 the latest Android version available?

**Comment by splosions** [comment]
I don't know how to see if version 5.1.1 is the only version, but I don't want to touch it because I don't want to end up erasing something important from my phone.
So that means the problem is something to do with my cell phone?

I finally managed to put some mods (one of optimization that changed absolutely nothing and still very slow in cities and one of better faces), I tried to use an app called apk editor and yet I didn't find any cfg, maybe the name is different or I just can't find.
I think it's something similar to what is in this forum https://forum.openmw.org/viewtopic.php?t=5440
he game is not unplayable but I still want the water:(
